### PR TITLE
Cloud App Security article additional edits

### PR DESCRIPTION
**Pull request by dankad256**

### CODE REVIEW

**Comment by PRMerger16** [comment]
@dankad256 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: powerbi-docs/admin/service-security-using-microsoft-cloud-app-security-controls.md [hunk 1]
```diff
@@ -14,25 +14,25 @@ LocalizationGroup: Data from files
...
 
-### Cloud App Security licensing
+## Configure real-time controls for Power BI with Cloud App Security
 
-In order to have Cloud App Security in your tenant, you must have one of the following [licenses](https://query.prod.cms.rt.microsoft.com/cms/api/am/binary/RE2NXYO):
-* Microsoft Cloud App Security: Provides Cloud App Security capabilities for all supported apps, part of the EMS E5 and Microsoft 365 E5 suites.
-* Office 365 Cloud App Security: Provides Cloud App Security capabilities only for Office 365, part of the Office 365 E5 suite.
-* Azure Active Directory Premium P1, in order to benefit from the major Cloud App Security capabilities.
+> [!NOTE]
+> * Azure Active Directory Premium P1 license is required in order to benefit from Cloud App Security real-time controls.
```
Comment (paulinbar) [inline powerbi-docs/admin/service-security-using-microsoft-cloud-app-security-controls.md:33]: An Azure Active Directory Premium P1 license...

File: powerbi-docs/admin/service-security-using-microsoft-cloud-app-security-controls.md [hunk 2]
```diff
@@ -57,6 +57,20 @@ Sensitivity labels enable you to classify and help protect sensitive content, so
 
 You can read the article on [sensitivity labels in Power BI](service-security-sensitivity-label-overview.md), which goes into detail about the process of using sensitivity labels for Power BI. See below for an [example of a Power BI policy based on sensitivity labels](#example).
 
+## Custom policies to alert on suspicious user activity in Power BI
+
+Cloud App Security activity policy enable administrators to define their own custom rules, to help detect user behavior that deviates from the norm, and even possibly act upon it automatically, if it seems too dangerous. For example:
```
Comment (paulinbar) [inline powerbi-docs/admin/service-security-using-microsoft-cloud-app-security-controls.md:62]: either 
A Cloud App Security activity policy enables administrators to define their own custom rules to help detect user behavior ...
or
Cloud App Security activity policies enable administrators to define their own custom rules to help detect user behavior .....

File: powerbi-docs/admin/service-security-using-microsoft-cloud-app-security-controls.md [hunk 3]
```diff
@@ -57,6 +57,20 @@ Sensitivity labels enable you to classify and help protect sensitive content, so
...
+
+* **Massive sensitivity label removal.** For example: alert me when sensitivity labels are removed by a single user from 20 different reports in a time window shorter than 5 minutes.
+
+* **Encrypting sensitivity label downgrade.** For example: alert me when a report that was with the ‘Highly confidential’ sensitivity label is now classified as ‘Public’.
```
Comment (paulinbar) [inline powerbi-docs/admin/service-security-using-microsoft-cloud-app-security-controls.md:66]: when a report that was with the -> when a report that had a

**Review by paulinbar — CHANGES_REQUESTED** [review]

**Review by paulinbar — APPROVED** [review]

**Comment by PRMerger14** [comment]
@dankad256 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by paulinbar — APPROVED** [review]

**Pull request merged**